src/types/resources/itemImage.d.ts

```diff
@@ -9,6 +9,7 @@ declare module 'resources' {
         hash: string;
         createdAt: Date;
         updatedAt: Date;
+        featuredImg: boolean;
```

Comment:
the interface is allready called ItemImage, so we dont need to repeat the information in the parameter name: rename featuredImg to just featured.